keyboards/ploopyco/trackball_nano/keymaps/default/keymap.c

```diff
@@ -20,3 +20,12 @@
 
 // Dummy
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {{{ KC_NO }}};
+
+void suspend_power_down_user(void) {
+    // Switch off sensor + LED making trackball unable to wake host
+    adns5050_power_down();
+}
+
+void suspend_wakeup_init_user(void) {
+    adns5050_init();
```

Comment:
Is it necessary to reinit on wake?

Reply:
I think so: 
From page 16 of https://media.digikey.com/pdf/Data%20Sheets/Avago%20PDFs/ADNS-5050.pdf
> There are 2 ways to exit power down, either assert low NRESET pin or by writing 0x5a to Register 0x3a. A full reset will thus be executed.

Reply:
You missed the most important part of that paragraph: 
>  Any register settings must then be reloaded.

So yeah, it sounds like a re-init would be appropriate in this case.

I would say pull out the register init into a separate function, but this sensor doesn't run i2c/spi/uart init, so should be fine, as is, I think.